python/pyspark/ml/tests.py

```diff
@@ -371,6 +378,126 @@ def test_fit_maximize_metric(self):
         self.assertEqual(1.0, bestModelMetric, "Best model has R-squared of 1")
 
 
+class RegressorTest(PySparkTestCase):
```

Comment:
Could we test ```numFeatures``` at each model that we can save cost of creating DataFrame and training model.